packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -172,6 +183,27 @@ export class TabBarRenderer extends TabBar.Renderer {
         return h.div({ className: 'p-TabBar-tabLabel', style }, data.title.label);
     }
 
+    protected getDecorations(tab: string): WidgetDecoration.Data[] {
```

Comment:
Should we document the methods:
- `getDecorations`
- `getDecorationData`
- `getIconClass`